plugins/inventory/terraform_state.py

```diff
@@ -36,6 +36,28 @@
       - Whether to include resources from Terraform child modules.
     type: bool
     default: false
+  include_providers:
+    description:
+      - A list of providers short names to include.
+      - Any instances from one of the providers will be included in the inventory.
+    type: list
+    elements: str
+    default: []
+    choices:
+      - google
+      - aws
+      - azurerm
+  exclude_providers:
+    description:
+      - A list of providers short names to exclude.
+      - Any instances from one of the providers will be excluded from the inventory.
+    type: list
+    elements: str
+    default: []
+    choices:
+      - google
+      - aws
+      - azurerm
```

Comment:
The relationship between these two options is a little weird and I'm trying to understand the use cases for having both of these. Is there a case where you would use both of these at the same time?

Reply:
@gravesm this is something similar to the `amazon.aws.ec2` inventory plugin. I am not sure there is a relevant use case using both options, most of the time one of them should be used

Reply:
For now I have just removed these 2 options, the inventory will basically returned all available instances